src/histolab/tiler.py

```diff
@@ -90,25 +92,39 @@ def locate_tiles(
             The alpha level to be applied to the rescaled slide, default to 128.
         outline: str
             The outline color for the tile annotations, default to 'red'.
+        linewidth: int
+            Thickness of line used to draw tiles
+        tiles: Iterable
+            Tiles to visualize. Optional, will be extracted if None.
 
         Returns
         -------
         PIL.Image.Image
             PIL Image of the rescaled slide with the extracted tiles outlined
         """
         img = slide.scaled_image(scale_factor)
-        img.putalpha(alpha)
+        if alpha is not None:
+            img.putalpha(alpha)
         draw = PIL.ImageDraw.Draw(img)
 
-        tiles = (
-            self._tiles_generator(slide, extraction_mask)[0]
-            if isinstance(self, ScoreTiler)
-            else self._tiles_generator(slide, extraction_mask)
-        )
-        tiles_coords = (tile[1] for tile in tiles)
-        for coords in tiles_coords:
+        if tiles is None:
+            tiles = (
+                self._tiles_generator(slide, extraction_mask)[0]
+                if isinstance(self, ScoreTiler)
+                else self._tiles_generator(slide, extraction_mask)
+            )
+        tiles_coords = [tile[1] for tile in tiles]
+
+        # maybe user specified colors for each tile
```

Comment:
comments like this are not meaningful in my opinion. We should use comments for specify something that could be not clear in the future. So in this case the comment should be removed if the statement or the code portion is clear enough, otherwise the statement has to be improved XD